### ISSUE TITLE
 Call to undefined function get_debug_type()

### ISSUE THREAD

**Issue by pitsolu** [bobthecow/psysh#690]
psysh version v0.10.8 phar
php 7.4.3

Hi,

Seems psysh is unable to see anonymous classes within abstractions

Example:
```

abstract class Message{

   function getMessages(){

      return new class(static::$messages) extends SomeAbstractArrayCls{ 

         public function __construct(&$messages) ...
```

**Comment by bobthecow** [comment]
Hi! Can you give me a working example of the bug? Thanks 🙂

**Comment by GrahamCampbell** [comment]
I think this is probably the same issue as https://github.com/bobthecow/psysh/issues/686.

**Comment by pitsolu** [comment]
> Hi! Can you give me a working example of the bug? Thanks slightly_smiling_face

```
//test.php
<?php

abstract class SomeAbstractArrayCls{

	protected $arr;

	public function getLast(){

		$item = end($this->$arr);

		return $item;
	}
}

abstract class Message{

	private static $messages;

   	public static function getMessages(){

    	return new class(static::$messages) extends SomeAbstractArrayCls{ 

        	public function __construct(&$messages){

        		$this->arr = $messages;
         	}
        };
    }

    public function addMessage($message){

    	static::$messages[] = $message;
    }
}

class Err extends Message{

	private $message;

	public function __construct(string $message){

		$this->message = $this->addMessage($message);
	}

	public function __toString(){

		return $this->message;
	}
}
```

To test:

```
>>> require "test.php"
=> 1
>>> new Err("error 401")
=> Err {#2570}
>>> Err::getMessages()
PHP Error:  Call to undefined function get_debug_type() in phar:///opt/bin/psysh/bin/psysh/vendor/symfony/var-dumper/Cloner/AbstractCloner.php on line 73
```

**Comment by pitsolu** [comment]
> I think this is probably the same issue as #686.

Btw am using using php 7.4.3

**Comment by bobthecow** [comment]
Good call @GrahamCampbell, looks like that's the one!

@pitsolu for now, installing via composer rather than using one of our pre-bundled phar releases will work around this.

**Comment by pitsolu** [comment]
Okay, thanks. 

Great tool you have here guys! @bobthecow @GrahamCampbell

**Comment by bobthecow** [comment]
This has been resolved in v0.10.11!
